Configurateur/front/test/container-creation/container_creation_test.dart

```diff
@@ -118,120 +118,6 @@ void main() {
     await tester.tap(find.byKey(const Key('terminate')));
   });
 
-  testWidgets('Container Creation back view panel from front',
```

Comment:
Why deleting so much tests ?

Reply:
Don't exist anymore on the screen with the new interface so I remove it